01.fizzbuzz/fizzbuzz.rb

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env ruby
+
+numbers = (1..20).to_a
+
+numbers.each do |number|
```

Comment:
```suggestion
(1..20).each do |number|
```

配列にしなくても直接rangeオブジェクトをループさせることができます。